setup.py

```diff
@@ -33,15 +37,45 @@ def run(self):
         develop.run(self)
 
 
-install_requires = ["networkx"]  # alternatively, read from `requirements.txt`
+# alternatively, read from `requirements.txt`
+install_requires = ["networkx", "setuptools"]
 extra_requires = ["pandas"]  # optional dependencies
 test_requires = ["pytest"]  # test dependencies
 dev_requires = ["pre-commit"]  # dev dependencies
 
 setup(
-    name="ti-edge-ai-demos",
-    version="0.1.0",
+    name=name,
+    version=version,
+    description='RidgeRun Edge AI Demos',
+    long_description='RidgeRun custom demos from TI Computer Vision and Deep Learning SDKs',
+    long_description_content_type='text/markdown',
+    author=author,
+    author_email='support@ridgerun.com',
+
+    packages=find_packages(
+        exclude=[
+            "*.tests",
+            "*.tests.*",
+            "tests.*",
+            "tests"]),
+    scripts=[
+        'main.py',
```

Comment:
is `main.py` located in same direction than `setup.py`?

Reply:
Yes, but just for this PR. In the other PR regarding the demo stuff, I add it to bin/, therefore I'll come back to this file and adding it accordingly